index.html

```diff
@@ -46,20 +47,177 @@ <h1 class="headline-title">I'm Uzair.<br>Glad to see you!</h1>
+                    </p>
+                    <a class="social-title" href="index.html">LET'S CONNECT</a>
+                    <ul class="social-links">
+                        <li class="social-links">
+                            <a href="#">
+                                <img src="resources/icon/twitter.svg" alt="Twitter">
+                            </a>
+                        </li>
+                        <li class="social-links">
+                            <a href="#">
+                                <img src="resources/icon/linkedin icon.svg" alt="linkedin">
+                            </a>
+                        </li>
+                        <li class="social-links">
+                            <a href="#">
+                                <img src="resources/icon/github.svg" alt="github">
+                            </a>
+                        </li>
+                        <li class="social-links">
+                            <a href="#">
+                                <img src="resources/icon/Vector.svg" alt="vector">
+                            </a>
+                        </li>
+                        <li class="social-links">
+                            <a href="#">
+                                <img src="resources/icon/VectorM.svg" alt="vectorm">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="action">
+                    <a class="btn" href="#">Get my resume</a>
+                </div>
+            </div>
+            <div class="framework">
+                <ul class="about-ul">
+                    <h3 class="sub-title">Languages <img class="down-arrow-icon" src="./resources/icon/down arrow.png" alt="down"></h3>
+                    <div class="framework-items">
+                        <li class="ul-item">
+                            <img src="./resources/icon/Ellipse 1.png" alt="javascript">
+                            <span>JavaScript</span>
+                        </li>
+                        <li class="ul-item">
+                            <img src="./resources/icon/Ellipse 2.png" alt="html">
+                            <span>HTML</span>
+                        </li>
+                        <li class="ul-item">
+                            <img src="./resources/icon/Ellipse 3.png" alt="CSS">
+                            <span>CSS</span>
+                        </li>
+                    </div>
+                </ul>
+                <ul class="about-ul">
+                    <h3 class="sub-title">Frameworks <img class="arrow-icon" src="./resources/icon/side arrow.png" alt="down"></h3>
+                </ul>
+                <ul class="about-ul">
+                    <h3 class="sub-title">Skills <img class="arrow-icon" src="./resources/icon/side arrow.png" alt="down"></h3>
+                </ul>
```

Comment:
- [ ] The `ul` tag is like a container for the `li` tags in other words its a container tag to group` li` elements. So I would suggest you to not use it to enclose `h3` elements. However you can use `<li>` tags instead of `<ul>` tag and replace the `div` container tag with `ul` tags For example:

```html
<ul class="framework">
                    <li>
                        <h3>Languages</h3>
                       <img class="down-arrow-icon" src="./resources/icon/down arrow.png" alt="down">
                        <ul class="framework-items>
                            <li>
                                <img src="image/final/Frame 9.png" alt="" />
                            </li>
                            <li>
                                <img src="image/final/Frame 10.png" alt="" />
                            </li>
                            <li>
                                <img src="image/final/Frame 11.png" alt="" />
                            </li>
                        </ul>
                    </li>
                    <li>
                     <h3>Skills</h3>
                    <img class="arrow-icon" src="./resources/icon/side arrow.png" alt="down">
                   </li>
                     <li>
                     <h3>Frameworks</h3>
                    <img class="arrow-icon" src="./resources/icon/side arrow.png" alt="down">
                   </li>
                </ul>
```